src/aosm/azext_aosm/configuration_models/onboarding_vnf_input_config.py

```diff
@@ -231,6 +231,11 @@ def validate(self):
             raise ValidationError("arm_template must be set")
         if not self.images:
             raise ValidationError("You must include at least one image")
+        for image in self.images:
+            (_, _, version) = split_image_path(image)
+            if not is_semver(version):
+                raise ValidationError(f"{image} has invalid version '{version}'.\n"
+                                      "Allowed formats are major.minor.patch")

```

Comment:
This is not what the semver regex matches. We should be clear what we allow. Do we actually support semver, or must it be `<int>.<int>.<int>`.

Fixing this involves updating the regex and/or this error message.